maquetado/scss/_footer.scss

```diff
@@ -0,0 +1,40 @@
+.footer {
+  align-items: center;
+  background-color: $blue;
+  color: $white;
+  display: flex;
+  flex-direction: column;
+}
+
+.foo-brand {
+  color: $white;
+  font-size: 15px;
+  margin: 40px 0 20px;
+  text-transform: uppercase;
+}
+
+.foo-logo {
+  margin-bottom: 15px;
+}
+
+.foo-final {
+  align-self: flex-end;
+  color: $white;
+  margin: 0 25px 30px 0;
```

Comment:
I think that isn't necessary the margin here, set `padding` property to `.footer` class